### ISSUE TITLE
Anaconda is not opening/working

### ISSUE THREAD

**Issue by beginner21** [ContinuumIO/anaconda-issues#11879]
### Actual Behavior
--> I have been using Anaconda for past 8 months, for past few weeks I haven't used Anaconda.
--> But this week, when I tried to open Anaconda, nothing opens. 
--> Even tried uninstalling and re-install the latest version, but nothing works.
--> Kindly provide the solution as soon as possible.

### Expected Behavior
--> When I launch Anaconda Navigator, it should open and am here to use JupyterLab and Jupyter Notebook, everything should work appropriately.

##### Operating System:  Windows 10

**Comment by d8970** [comment]
I am also have a similar issue . However I am using Mac OS Catalina Version 10.15.5 (19F101)
It used to launch when I press command + space and enter anaconda-Navigator there..

I get can't be opened
<img width="430" alt="Screen Shot 2020-06-17 at 4 07 05 PM" src="https://user-images.githubusercontent.com/26498756/84959696-b675d980-b0b4-11ea-9df7-fc57a9de9302.png">

**Comment by d8970** [comment]
Fixed found Anaconda-Navigator under my home folder or ~/opt/anaconda3 and was able to launch from there...

**Comment by meRscliche** [comment]
(1) I encounter exactly the same problem as you did on my NB(Win10 & Anaconda 2020.07) today.
After googling, somehow I managed to solve it, but I got to launch Jupyter and Spyder in Anaconda Navigator from now on. Well, it's better than never launching Jupyter and Spyder.

(2) Launch the Anaconda Prompt as a system administrator, then enter the following line in order:
```
conda update conda
conda update --all
conda remove QtPy PyQt5
pip install QtPy PyQt5
```
ERROR: spyder 4.1.4 requires pyqtwebengine<5.13; python_version >= "3", which is not installed.
ERROR: spyder 4.1.4 has requirement pyqt5<5.13; python_version >= "3", but you'll have pyqt5 5.15.1 which is incompatible.
_The errors above are just warnings and won't stop the process._
```
conda update anaconda-navigator
anaconda-navigator --reset
conda update anaconda-client
conda update -f anaconda-client
```
